Goobi/src/de/sub/goobi/export/dms/AutomaticDmsExportWithoutHibernate.java

```diff
@@ -138,9 +138,6 @@ public boolean startExport(ProcessObject process) throws DAOException, IOExcepti
 				break;
 
 			case METS_AND_RDF:
```

Comment:
I did not remove it because it matches the action taken here (`new RDFFile(this.myPrefs)`). The correct solution would be to name all cases which need this action and to throw an exception in the default case (which should never occur). But that solution would require more code reviews and good testing.
Therefore I suggest to apply the PR as it is.